sqlmesh/schedulers/airflow/mwaa_client.py

```diff
@@ -31,7 +31,8 @@ def set_variable(self, key: str, value: str) -> t.Tuple[str, str]:
         Returns:
             A tuple of stdout and stderr from the MWAA CLI.
         """
-        return self._post(f"variables set {key} '{value}'")
+        value=value.replace('\\', '\\\\').replace('"', '\\"')
```

Comment:
what encoding is this? why do you have to do this manually? should you just call json on it?

Reply:
so value = json.dumps(value)

Reply:
That's actually a great point! So it can be simplified to:
```
return self._post(f"variables set {key} {json.dumps(value)}")
```
Good catch 👍

Reply:
Reference example confirming it works as expected:
```
In [8]: value = "{\"test\": \"value\n\"}"

In [9]: print(f"variables set test_key {json.dumps(value)}")
variables set test_key "{\"test\": \"value\n\"}"
```

Reply:
@tobymao on the other hand I'm not sure this is enough. We still need to escape `\`, which `json.dumps` doesn't take care of.

Reply:
we'll handle it from, thanks for the PR!

Reply:
@tobymao Does `we'll handle it from here` mean to close this PR and let you open a new one?

Reply:
no, @izeigerman will merge this and explore